stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1795,3 +1795,35 @@ function normalize(a::AbstractArray, p::Real = 2)
         return T[]
     end
 end
+
+"""
+    islc(A::AbstractMatrix, y::AbstractVector)
+
+For a matrix A and a vector y, checks if y is a linear combination of columns of A.
+In other words, checks that equation Ax = y has a solution or not.
+
+
+# Examples
+```jldoctest
+julia> A = [1 1; 1 0]
+2×2 Matrix{Int64}:
+ 1  1
+ 1  0
+julia> y = [3, 1]
+ 2-element Vector{Int64}:
+  3
+  1
+julia> println(islc(A, y))
+  true
+```
+"""
+function islc(A::AbstractMatrix, y::AbstractVector)
+    tA = copy(A)
```

Comment:
Taking the copies seem unnecessary, the arrays are not mutated?

Reply:
Hey @baggepinnen,
Thanks for the response and advice,
About the name, I was struggling with the naming and couldn't find a standard about how to name a method, I'd appreciate it if you could provide me with a good name for this method.
And about the copies, i'll be fixing it for the next commit,
Thanks in advance.